components/shopify_developer_app/README.md

```diff
@@ -0,0 +1,31 @@
+# Overview
+By creating a custom app on Shopify, you will be able to configure the exact scopes that you require to build the workflows that you need.
+
+# Getting Started
+
+To get started, you will need to create a custom Shopify app in the Shopify Admin UI. The steps are outlined below:
+
+## Create a Shopify app
+1. Sign in to your Shopify admin account, and navigate to the store that you are managing. 
+2. Click **Develop apps**.
+3. Click **Allow custom app development**.
+4. Click **Create an App**, and name the app "Pipedream"
```

Comment:
they can name it whatever they want, right? there's probably also a world in which people have existing oauth apps they want to use in Pipedream, right?

Reply:
I don't have a strong preference on naming, but was mostly following the Gmail (Developer App) and other examples where that's what we have written.

With regard to other OAuth apps, this would only be the case if they created their app via the CLI, but I think the best path forward for supporting this is to go the access token route.

Reply:
yea, with non-oauth apps I agree users will likely create Pipedream-specific tokens, so this lgtm